ios/src/TGGeoPolygon.mm

```diff
@@ -0,0 +1,80 @@
+//
+//  TGGeoPolygon.mm
+//  tangram
+//
+//  Created by Karim Naaji on 10/27/16.
+//
+//
+
+#import "TGGeoPolygon.h"
+
+#include <vector>
+
+@interface TGGeoPolygon () {
+    std::vector<TGGeoPoint> coordinates;
+    std::vector<int> rings;
```

Comment:
@msmollin Out of curiosity, is there a CF/NS type that does the job of `std::vector` in Objective-C land? (i.e. a dynamic array with contiguous memory)